Tests/Marketplace/search_and_install_packs.py

```diff
@@ -138,9 +138,9 @@ def search_pack(client, prints_manager, pack_display_name):
 
             else:
                 print_msg = 'Did not find pack {} in bucket.\n'.format(pack_display_name)
-                prints_manager.add_print_job(print_msg, print_color, 0, LOG_COLORS.YELLOW)
+                prints_manager.add_print_job(print_msg, print_color, 0, LOG_COLORS.RED)
                 prints_manager.execute_thread_prints(0)
-                return {}
+                raise Exception(print_msg)
```

Comment:
I thin that if we will raise exception here this will not keep going and try search the other packs. correct?
we need the build to fail in the end, but we need it to keep going until the end.

Reply:
This is running inside a thread, so it will crash when we do `thread.join`. That specific thread will stop working so some packs might be missed. That's why it's a little complicated to know the state of this process. @guykeller do you have an idea how to handle this situation similarly to the nightly build?

Reply:
@yaakovi do you have an idea maybe?

Reply:
didn't find anything useful to handle it.
have you looked at `multiprocessing.Pool` docs instead of `multithreading.Thread`?

Reply:
@yaakovi I did, but that would take a lot of effort as a lot of the logic uses a thread index.